File: 07212023/46_165_Number_Compare/java/NumberCompare.java
public class NumberCompare {
    public static void main(String[] args) {
        System.out.println(numberCompare(1, 1));
        System.out.println(numberCompare(1, 0));
        System.out.println(numberCompare(2, 4));
    }

    public static String numberCompare(int a, int b) {
        if (a == b) {
            return "Numbers are equal";
        } else if (a > b) {
            return "First is greater";
        }

        return "Second is greater";
    }
}

File: 07212023/45_165_Number_Compare/python/number_compare.py
def number_compare(a: int, b: int) -> str:
    """Do a numbers comparison and return which number is greater or are they equal."""
    if a == b:
        return "Numbers are equal"
    elif a > b:
        return "First is greater"
    return "Second is greater"

print(number_compare.__doc__)
print(number_compare(1,1)) # "Numbers are equal"
print(number_compare(1,0)) # "First is greater"
print(number_compare(2,4)) # "Second is greater"